### PR TITLE
Added flag to ignore case and custom validator for new tags

### PR DESCRIPTION
**Pull request by Pe-te**
Added flag to ignore case and custom validator for new tags

### CODE REVIEW

File: Source/WSTagsField.swift [hunk 1]
```diff
@@ -326,8 +326,19 @@ open class WSTagsField: UIScrollView {
         addTag(WSTag(tag))
     }
 
+    open var validator: ((WSTag, [WSTag]) -> Bool)?
+
+    open var ignoreCaseWhenAdding: Bool = false
```
Comment (ricardopereira) [inline Source/WSTagsField.swift]: @Pe-te For the framework to be, at the same time, more straightforward and flexible, what to you think to just keep the `validator`?

Comment (ricardopereira) [reply Source/WSTagsField.swift]: Oh BTW, I think the `WSTagsField` has an event for that.

Comment (ricardopereira) [reply Source/WSTagsField.swift]: Those [`onShouldAcceptTag`](https://github.com/whitesmith/WSTagsField/blob/14338e327015da58166adb032e772efbc936d139/Source/WSTagsField.swift#L190) helps for what do you need?

Comment (ricardopereira) [reply Source/WSTagsField.swift]: Hmm, maybe not. Please remove `ignoreCaseWhenAdding` and rename `validator` to `onValidateTag` (put it in the Events section of the code). Thanks a lot!

**Comment by ricardopereira** [comment]
Add an example in the README if possible. Thank you so much for your contribution.

**Comment by Pe-te** [comment]
Yeah, sounds cleaner .. removed, renamed and readme! :)

Don't see a button to resubmit, so I guess you can still merge the new version now?

**Pull request merged**